tests/README.md

```diff
@@ -11,10 +11,15 @@ We are using the following modules and packages for our unit tests:
 - [unittest](https://docs.python.org/3/library/unittest.html) (standard library)
 - [unittest.mock](https://docs.python.org/3/library/unittest.mock.html) (standard library)
 - [coverage.py](https://coverage.readthedocs.io/en/stable/)
+- [pytest-cov](https://pytest-cov.readthedocs.io/en/latest/index.html)
+
+We also use the following package as a test runner:
+- [pytest](https://docs.pytest.org/en/6.2.x/)
 
 To ensure the results you obtain on your personal machine are comparable to those generated in the CI, please make sure to run your tests with the virtual environment defined by our [Poetry Project](/pyproject.toml). To run your tests with `poetry`, we've provided two "scripts" shortcuts:
 
-- `poetry run task test` will run `unittest` with `coverage.py`
+- `poetry run task fast-test` will run `pytest`.
+- `poetry run task test` will run `pytest` with `pytest-cov`.
```

Comment:
The line above says "two scripts" but it is four now.

Reply:
Or maybe just three. :smile:

Reply:
Counting is hard. e45843e